### ISSUE TITLE
Scraping an IEEE HTML view should grab the PDF too

### ISSUE THREAD

**Issue by LinuxMercedes** [LinuxMercedes/zotero#4]

**Comment by michaelwisely** [comment]
In FireFox, you can download a PDF to Zotero, and it'll scrape citation bits out of the PDF. That doesn't grab as much detail as the IEEEExplore record, though.

**Comment by LinuxMercedes** [comment]
Yeah, plus it's a few extra steps. If I scrape the IEEExplore abstract page, everything works fine; probably just need to tell zotero how to understand the html view page.

**Comment by michaelwisely** [comment]
Yeh. Something with [this guy](https://github.com/zotero/translators/blob/master/IEEE%20Xplore.js). I assume we can do something similar with [ACM DL](https://github.com/zotero/translators/blob/master/ACM.js), unless it already does it.
